src/main/java/frc/robot/commands/Intake.java

```diff
@@ -0,0 +1,28 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
+// NOTE:  Consider using this command inline, rather than writing a subclass.  For more
+// information, see:
+// https://docs.wpilib.org/en/latest/docs/software/commandbased/convenience-features.html
+public class Intake extends SequentialCommandGroup {
+  /**
+   * Creates a new Intake.
+   */
+  boolean setIntake = true;  // True is on, False is off
+  public Intake(Intake intake) {
+    addCommands(
+      // Turns on and off intake motor
+      new IntakeSet(setIntake)
+    );
+    // Add your commands in the super() call, e.g.
+    // super(new FooCommand(), new BarCommand());
+  }
+}
```

Comment:
I'm going to be honest, I didn't think about the naming conflicts here... For now, please delete this, I'm not sure we even need it, sorry.